src/SoundModule.py

```diff
@@ -0,0 +1,235 @@
+"""
+The SoundModule is a library of functions designed to perform real-time signal processing on blocks of audio.
+Each function takes inputs and outputs in a similar fashion, hence allowing them to
+be aggregated into a signal chain in an ad-hoc fashion.
+The various functions currently available in Sound-Module were developed keeping in mind
+their use in building the aesthetics necessary for SERIAL.
+https://github.com/reckoner165/TraceMelody
+
+Sumanth Srinivasan - December 2016
+"""
+
+import math
```

Comment:
All `math` imports could be clubbed together as -
`from math import cos, sin, pi, ceil, floor, isnan` (please add other methods that I have missed).

Reply:
Hi @reckoner165. You needn't club them together in this pull request. We will do it as a separate one as a lot of imports need to be clubbed together. Sorry for the confusion. 😅